src/components/__snapshots__/CircleButton.test.tsx.snap

```diff
@@ -66,11 +66,7 @@ exports[`CircleButton renders correctly with minimum props 1`] = `
       xmlns="http://www.w3.org/2000/svg"
     >
       <path
-        d="M2.141,89.13c1.425,1.429,3.299,2.142,5.167,2.142c1.869,0,3.742-0.713,5.167-2.142l33.591-33.592L79.657,89.13
-          c1.426,1.429,3.299,2.142,5.167,2.142c1.867,0,3.74-0.713,5.167-2.142c2.854-2.854,2.854-7.48,0-10.334L56.398,45.205
```

Comment:
these three snaps are the same, just different indentations.

Reply:
It was this which isn't needed. They'll go back to the way they were :) https://github.com/valora-inc/wallet/pull/3961/files#r1270625689